source/trunk/Esf.WebApi/Esf.WebApi.csproj

```diff
@@ -0,0 +1,276 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Esf.WebApi is good name as log as we don't associate it with the concrete microsoft technology. If we use this I'll like to use it as an internal term for Http Service or other cross machine service.